UiPath/Script.Activities
# OSAC 35 - added tags to nuspec and csproj files

Description
Add "UiPathActivities" tag to all open source activities packs
How to test
Add new package into UiPath studio and check for the Tags attribute
Jira
[OSAC-35](https://uipath.atlassian.net/browse/OSAC-35)
Label:
24S
